### DISCUSSION TITLE
Play audio clip between songs (like radio branding)

### DISCUSSION THREAD

**Discussion by Stoutscientist** [AzuraCast/AzuraCast#7111] [Q&A] [answered]
On normal radio stations, between most songs they have a small branding theme they play. Is there any way to do this on AzuraCast?

**Comment by silvaalex2021** [comment, marked as answer]
Right now, what works for me is to put the clips I want played between songs in a separate playlist, set this playlist to "Jingle mode" and shuffle it, and then set this playlist to play once for each 1 song. It works quite well!

**Reply by Stoutscientist** [reply]
Perfect, thanks!
